### DISCUSSION TITLE
Why did I submit a student certification and it was rejected?

### DISCUSSION THREAD

**Discussion by dazzlingcc** [community/community#56292] [GitHub Education]
### Hi everyone,

I filled out my information as required, but when selecting my school on the map, I found that it was not an option, so I don't know if it was rejected for this reason.

**Comment by samus-aran** [comment]
Hi @dazzlingcc 👋 

If your school is not available to select on the map you can proceed to supply it's details. If you can not find your school we wouldn't recommending selection another school and it will then use that school's information to cross-reference your evidence which would be inaccurate.

Let us know how you get on 👍
